pkg/systemd/init.js

```diff
@@ -652,6 +652,14 @@ $(function() {
                                            UnitButton: unit_action_btn,
                                            FileButton: file_action_btn,
                                            NotMetConditions: not_met_conditions,
+                                           Relationships: [
+                                             { Name: _("Requires"),       Units: cur_unit.Requires },
+                                             { Name: _("Wants"),          Units: cur_unit.Wants },
+                                             { Name: _("Wanted by"),      Units: cur_unit.WantedBy },
```

Comment:
Nitpick: This should perhaps be "Wanted By" (Title Case), or we consider these fixed identifiers, use `WantedBy` and not translate them at all? @andreasn, WDYT? I wouldn't mind the latter, as translating them would lose the connection to the actual identifiers in `.service` files; but then again we translate a lot of similar things in Cockpit, including the unit status.

Reply:
I have an irrational distaste for English title-case. :)

I guess this is another question for designers to settle....